src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -79,6 +79,7 @@ public class ViewResult : ActionResult
                             actionContext = context,
                             result = this,
                             viewName = viewName,
+                            viewEngineType = viewEngine.GetType(),
```

Comment:
Yep, see that... So the next point past this was to see if we can get the `ViewEngineResult` to include `IViewEngine` or if there is a better way all together. Any thoughts?